packages/smooth_app/lib/pages/personalized_ranking_model.dart

```diff
@@ -0,0 +1,104 @@
+import 'package:flutter/material.dart';
+import 'package:openfoodfacts/openfoodfacts.dart';
+import 'package:smooth_app/data_models/product_preferences.dart';
+import 'package:smooth_app/data_models/up_to_date_product_provider.dart';
+import 'package:smooth_app/database/dao_product.dart';
+import 'package:smooth_app/database/local_database.dart';
+import 'package:smooth_app/pages/tmp_matched_product_v2.dart';
+
+enum LoadingStatus {
+  LOADING,
+  LOADED,
+  ERROR,
+}
+
+/// Model that computes the scores and sorts the barcodes accordingly.
+class PersonalizedRankingModel with ChangeNotifier {
```

Comment:
Don't know the exact name of the off-dart version but maybe Smooth before it so that it's explict

Reply:
There are no "models" in off-dart, and no `ChangeNotifier` over there either. No possible confusion.
Besides, if we put `Smooth` everywhere it's going to look like [Smurf language](https://en.wikipedia.org/wiki/The_Smurfs#Language) ;)